.env.defaults

```diff
@@ -51,15 +51,15 @@ TWITTER_ACCOUNT=OdyseeTeam
 BRANDED_SITE=odysee
 
 # IMAGE ASSETS
-YRBL_HAPPY_IMG_URL=https://spee.ch/spaceman-happy:a.png?quality=85&height=457&width=457
-YRBL_SAD_IMG_URL=https://spee.ch/spaceman-sad:d.png?quality=85&height=457&width=457
-LOGIN_IMG_URL=https://player.odycdn.com/speech/odysee-sign-up:d.png
-LOGO=https://spee.ch/odysee-logo-png:3.png?quality=85&height=200&width=200
-LOGO_WHITE_TEXT=https://spee.ch/odysee-white-png:f.png?quality=85&height=300&width=1000
-LOGO_DARK_TEXT=https://spee.ch/odysee-png:2.png?quality=85&height=300&width=1000
-AVATAR_DEFAULT=https://spee.ch/spaceman-png:2.png?quality=85&height=180&width=180
-MISSING_THUMB_DEFAULT=https://spee.ch/missing-thumb-png?quality=85&height=390&width=220
-FAVICON=https://spee.ch/favicon-png:c.png
+YRBL_HAPPY_IMG_URL=https://thumbnails.odycdn.com/optimize/s:300:300/quality:85/plain/https://spee.ch/spaceman-happy:a.png
+YRBL_SAD_IMG_URL=https://thumbnails.odycdn.com/optimize/s:300:300/quality:85/plain/https://spee.ch/spaceman-sad:d.png
+LOGIN_IMG_URL=https://thumbnails.odycdn.com/optimize/s:400:507/quality:85/plain/https://player.odycdn.com/speech/odysee-sign-up:d.png
+LOGO=https://thumbnails.odycdn.com/optimize/s:100:100/quality:85/plain/https://spee.ch/odysee-logo-png:3.png
+LOGO_WHITE_TEXT=https://thumbnails.odycdn.com/optimize/s:1000:300/quality:85/plain/https://spee.ch/odysee-white-png:f.png
+LOGO_DARK_TEXT=https://thumbnails.odycdn.com/optimize/s:1000:300/quality:85/plain/https://spee.ch/odysee-png:2.png
+AVATAR_DEFAULT=https://thumbnails.odycdn.com/optimize/s:160:160/quality:85/plain/https://spee.ch/spaceman-png:2.png
+MISSING_THUMB_DEFAULT=https://thumbnails.odycdn.com/optimize/s:390:220/quality:85/plain/https://spee.ch/missing-thumb-png
+FAVICON=https://thumbnails.odycdn.com/optimize/s:100:100/quality:85/plain/https://spee.ch/favicon-png:c.png
```

Comment:
Per Google's guideline, maybe use 144x144?

https://developers.google.com/search/docs/advanced/appearance/favicon-in-search?hl=en&visit_id=637834228203607550-4028388767&rd=1#guidelines

> _Your favicon must be a multiple of 48px square, for example: 48x48px, 96x96px, 144x144px and so on. SVG files don't have a specific size._

Reply:
Good call, will update before shipping.

Reply:
(it was 100 before also, so this is a good fix).

Will check if we have a higher quality version somewhere.

Reply:
Note that here is also https://odysee.com/public/favicon-spaceman.png which is exactly 96x96.

It was added to satisfy the following guideline:
<img width="624" alt="image" src="https://user-images.githubusercontent.com/64950861/159205734-1d453cd8-0b86-4ae3-bba1-6fa435cdc1c4.png">

Maybe consolidate all favicons to use that size and domain?